internal/command/testdata/rspec/Gemfile

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "rspec"
```

Comment:
The new test in `internal/command/run_test.go` calls `rspec --dry-run` during the plan generation, so needs a `Gemfile` to be present.